zuul.d/jobs.yaml

```diff
@@ -52,86 +52,145 @@
     name: terraform-provider-openstack-unittest
     parent: golang-test
     description: |
-      Run terraform provider openstack unit test
+      Run terraform-provider-openstack unit test
     run: playbooks/terraform-provider-openstack-unittest/run.yaml
     nodeset: ubuntu-xenial-ut
 
+## ========== terraform-provider-openstack + openstack jobs ==============
+
+### parent job
 - job:
     name: terraform-provider-openstack-acceptance-test
     parent: golang-test
     description: |
-      Run terraform provider openstack acceptance test on master branch
+      Run terraform-provider-openstack acceptance test against with master OpenStack
```

Comment:
`against` and `with` are two preposition words, please only use one.

Reply:
sure, done.